src/pages/ConnectWallet/web3Utils.ts

```diff
@@ -18,12 +18,14 @@ export enum NetworkChainId {
   'Mainnet' = 1,
   'Sepolia' = 11155111,
   'Holesky' = 17000,
```

Comment:
Can open a separate PR to formally deprecate Holesky

Reply:
Yeah, I agree this is the right way to do that